code/modules/mob/living/carbon/human/human.dm

```diff
@@ -923,7 +923,7 @@
 	return ..()
 
 /mob/living/carbon/human/is_literate()
```

Comment:
Please add appropriate documentation to this proc.

Reply:
This isn't where the proc is defined though, it's defined on `/mob`, don't only the original definitions need documentation?

Reply:
Never mind then, I'm dumb.